src/cmake/ac_macros.cmake

```diff
@@ -5,6 +5,18 @@ MACRO(AC_ADD_SCRIPT path)
     CU_ADD_GLOBAL("AC_SCRIPTS_SOURCES" "${path}")
 ENDMACRO()
 
+#
+# AC_ADD_SCRIPTS
```

Comment:
Just as a reminder for other devs: this can be used by modules to automatically load game scripts from a gived folder inside the **scripts_STAT_SRCS** array (without manually add them one by one).

Winfy could you give me edit permissions here, thus I can add the description of what it does inside the cmake comments?

Reply:
yes, sec